backend/app/services/chatbot_service.py

```
@@ -287,10 +289,10 @@ async def delete_chatbot(self, user_id: ObjectId, chatbot_id: str):
                 f"Chatbot with ID {chatbot_id} found"
             )
             
-            logger.debug(f"Removing chatbot from database")
+            logger.debug("Removing chatbot from database")
             result = await chatbot_crud.delete_chatbot(chatbot_id)
```

Ruff (0.8.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)